src/azul/project/hca/transformers.py

```diff
@@ -188,6 +189,8 @@ def _protocol_dict(protocol: api.Protocol) -> JSON:
         protocol_dict['paired_end'] = protocol.paired_end
     elif isinstance(protocol, api.AnalysisProtocol):
         protocol_dict['workflow'] = protocol.protocol_id
+    elif isinstance(protocol, api.ImagingProtocol):
+        protocol_dict['assay_type'] = [target.assay_type for target in protocol.target]
```

Comment:
```suggestion
        protocol_dict['assay_type'] = list(set(target.assay_type for target in protocol.target))
```

Reply:
A list is required here so we can keep duplicate values that will be counted up in the FrequencySetAccumulator.

Reply:
I see. Could we compress this by  

```suggestion
        protocol_dict['assay_type'] = dict(Counter(target.assay_type for target in protocol.target))
```

and handling dict values in `FrequencySetAccumulator.accumulate()`?

`Counter.update()` handles `dict` instances:

```python
>>> a = Counter([2])
>>> b = Counter([1,2])
>>> a.update(dict(b))
>>> a
Counter({2: 2, 1: 1})
```